project.yaml

```diff
@@ -13,3 +13,10 @@ actions:
     outputs:
       highly_sensitive:
         rows: output/query/rows.csv.gz
+
+  aggregate:
+    needs: [query]
+    run: python:latest python -m analysis.aggregate
```

Comment:
there's probably a good reason to write it this way, but why not:
`run: python:latest analysis/aggregate.py`

I assume this is because analysis/aggregate.py ends with:

```
if __name__ == "__main__":
    main()
```

Reply:
Nope! It's because we want to access `analysis.utils`, which means we want the _analysis_ directory to be on `PYTHONPATH`. Try cloning the repo, switching to the branch, and changing _project.yaml_.